docs/limitations.md

```diff
@@ -0,0 +1,61 @@
+
+## Expectations
+
+- Evolving UX, developer tools, and documentation
+- Frequent, sometimes breaking changes, but we will do our best to communicate them ahead of time in [Announcements](https://github.com/orgs/NillionNetwork/discussions)
+- Bugs - please report any you find in [Bugs](https://github.com/orgs/NillionNetwork/discussions/categories/bugs)
+- Missing features - suggest a feature request in [Ideas](https://github.com/orgs/NillionNetwork/discussions/categories/ideas)
+- Documentation gaps - if you notice something is missing, please let us know by creating a [Github Issue](https://github.com/NillionNetwork/nillion-docs/issues/new/choose)
+
+Here are some of the limitations you should be aware of when being an early builder with Nillion’s SDK.
+
+## Nillion Clients
+
+- We have released 3 clients to the community: a Python, JavaScript, and CLI Client. In general the clients have feature parity, however please refer to the [Nillion Client docs](http://localhost:3000/nillion-client) to see the exact functionality provided by each.
+- You can only compile programs from the CLI `pynadac` tool — compilation isn’t available in the Python or JavaScript clients.
+- If there is a particular feature you need and believe is missing in one of the clients, please report it in [Bugs](https://github.com/orgs/NillionNetwork/discussions/categories/bugs).
+
+### JavaScript Client
+
+- Currently the JS client is only tested on Chromium browsers (Chrome, Brave & Edge).
+- The JS client does not yet have SecretArray functionality.
+- The JS client is a browser client and does not yet work in NodeJs.
+
+### CLI Client
+
+Running the [`nillion-devnet`](/nillion-devnet) command (SDK tool) will spin up a local devnet on your machine.
+
+- The devnet that is spun up will be limited by the hardware it is running on. Keep an eye on the CPU usage when running large computations.
+- Pre-processing elements are generated from scratch each time a local devnet is spun up. This means you may need to wait a little time (10-15 seconds) before the network is able to store and compute. For reference, currently 8192 alphas & 8192 lambdas are generated when you spin up a new local devnet.
+- Anything you store (programs, secrets etc) in one instance of `nillion-devnet` will not be shared with another devnet you spin up, meaning you will have to restore any stored programs or secrets in the new devnet.
+
+### One node key per client
+
+- If you are instantiating multiple clients (users) at once (e.g. `Party1', `Party2`, `Party3` for a multi party compute), ensure that each instantiated client uses a different node key. If a node key is reused, across clients, you will see a timeout error.
+
+## Secrets
+
+- Secrets are not yet user scoped, so `store_id` values could collide and overwrite other secrets.
```

Comment:
store_id are uuids the probability of collision is almost none, other systems use them and they assume that they are not going to collide. Same happens when you generate a private key, the odds of colliding are so low that are ignored. Here a post explaining the odds of colliding https://jhall.io/archive/2021/05/19/what-are-the-odds/. After that, I will not document the possibilities of colliding.

Reply:
I'll remove the part about collision possibility (low) but secrets not being user scoped is worth keeping in since programs are user scoped.